src/components/settings/SettingsSSOTab/hooks.js

```diff
@@ -106,6 +106,13 @@ const useIdpState = () => {
         setCurrentError(
           ': Unable to verify provided metadata URL, please check your information and try again.',
         );
+      } else if (error.customAttributes?.httpErrorResponseData) {
+        const errorResponseData = JSON.parse(error.customAttributes.httpErrorResponseData);
```

Comment:
might be nicer to use a `409 - Conflict` error to signal the dupe rather than parsing an error message

https://developer.mozilla.org/en-US/docs/web/http/status/409